### ISSUE TITLE
Sending Syx files from Dexed does not work

### ISSUE THREAD

**Issue by rickyackel** [asb2m10/dexed#241]
First, Thank you for the great work on this.
The DX7 is called DX7 in Audio/MIDI Setup, if that means anything. When you right click (2 finger tap on my Touch Pad) on a file name, you get the option to send the file to the DX7. When I do this I get no activity on the MIDI Out activity lights on my midi man USB MIDISPORT 4x4. I downloaded Open Source Sysex Librarian and find that it sends the files fine. This is a problem in Stand Alone mode and also when imbedded in Logic Pro X.

**Comment by asb2m10** [comment]
In the PARM dialog, do you see your midi interface in the midi out combo box ?

**Comment by rickyackel** [comment]
Yes I do. I even see the “DX7” name of the port that is in Audio/MIDI setup.

**Comment by lfrancis** [comment]
I'm seeing the same thing on my M1 Pro MBP 32/1T Tahoe 26.6.1 using Dexed 1.0.1. Using Snoise MIDI Monitor I can see MIDI notes being sent via Dexed's keyboard but when I try to send sysex nothing is transmitted.

**Comment by lfrancis** [comment]
> I'm seeing the same thing on my M1 Pro MBP 32/1T Tahoe 26.6.1 using Dexed 1.0.1. Using Snoise MIDI Monitor I can see MIDI notes being sent via Dexed's keyboard but when I try to send sysex nothing is transmitted.

Figured it out -- I needed to configure an additional setting under the Parameters pane in addition to the MIDI routing under the Options settings.

<img width="914" height="748" alt="Image" src="https://github.com/user-attachments/assets/df49fc43-6df4-4749-bac1-020f07ddbcf6" />
